boot-speed/clouds/measure-cloud.py

```diff
@@ -323,6 +331,10 @@ def gen_archivename(metadata):
     arcname = cloud + "-" + inst_type + "-" + release + "_" + date
     return arcname
 
+def metaname2release(metaname):
```

Comment:
Have you looked at python3-distro-info?

```
import distro_info
    try:
        devel = distro_info.UbuntuDistroInfo().devel()
    except distro_info.DistroDataOutdated:
        devel = distro_info.UbuntuDistroInfo().stable()

    lts = distro_info.UbuntuDistroInfo().lts()
```

Reply:
Now I want again to implement it in pycludlib. It shouldn't be difficult really.

Reply:
Mmh no, I don't want to, as python3-distro-info is too Debian/Ubuntu specific, and I don't think we want the library to depend on it. It's not even in PyPI (`distroinfo` is a different thing).

Reply:
yeah for pycloudlib I would be -1 on it, but for our test scripts it can definitely include it.

Reply:
Done (a bit differently).